data/analytics/events.yml

```diff
@@ -303,14 +303,103 @@
         - name: method
         - name: link_domain
 
-- name: call out box
+- name: call out boxes
   events:
     - name: call out box link click
-      implemented: false
-      priority: medium
+      description: Link tracking for call to action, warning text, advisory callout, information callout, and content banner components.
+      tracker: link_tracker
+      implemented: true
+      priority: low
```

Comment:
Are there any examples you could add for call out boxes?

Reply:
Thanks, have added a banner example - we're not sure the warning text component ever contains links, so I don't have an example at the moment.